tot_model_suggester.py

```diff
@@ -0,0 +1,57 @@
+
+from model_suggester import ModelSuggester
+import guidance
+
+class ToTModelSuggester(ModelSuggester):       
+
+<<<<<<< HEAD
+    def _pairwise_relationship_program(self):
+=======
+    def pairwise_relationship_program(self):
+>>>>>>> validator
+        
+        '''
+        Uses tree of thought prompting to choose best answer for pairwise relationship
```

Comment:
Can you add a reference to the specific kind of ToT you implemented here?  

A key aspect of the original tree of thoughts papers is a recursive depth-first or breadth-first tree search, which isn't in this guidance program.  

But a later approach creates a single prompt version (no explicit search) inspired by ToT (see Tree of Thought prompting, https://github.com/dave1010/tree-of-thought-prompting) . What you are doing here seems to be more similar to this approach.  

I'd also suggest adding  a note / TODO comment about additional variations to experiment with.  E.g., Dave Hulbert's version includes asking multiple experts to make proposals and discuss.